django_server/API_DOCS.md

```
@@ -0,0 +1,719 @@
+---
+
+## 💬 Chats App
+
+### Stream Chat (Server-Sent Events)
+
+```http
+POST /api/chats/stream_chat/
+Authorization: Bearer <token>
+Content-Type: application/json
+
+{
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)